keyboards/xelus/pachi/pachi.c

```diff
@@ -0,0 +1,22 @@
+/* Copyright 2021 Harrison Chan (Xelus)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "pachi.h"
+
+void led_init_ports(void) {
+    palSetLineMode(LED_CAPS_LOCK_PIN, PAL_MODE_OUTPUT_PUSHPULL);
```

Comment:
Isn't this the default?

Reply:
Yeah, it is.
https://github.com/qmk/qmk_firmware/blob/e941ae38110c616884acbb7447350b150cdb9d4d/quantum/led.c#L105-L126

Reply:
Oh yep i'll just delete that